grails-app/views/taglib/_autoSuggest.gsp

```diff
@@ -43,9 +42,7 @@
                         add(items);
                     }));
             },
-            focus: function(event, ui) {
-                return false;
-            },
+            //focus: function( event, ui ) { event.preventDefault(); },
```

Comment:
Do we need to keep this?

Reply:
I was playing around with it and just forgot to remove it.